pkg/rbd/controllerserver.go

```diff
@@ -176,7 +182,12 @@ func (cs *ControllerServer) createBackingImage(rbdVol *rbdVolume, req *csi.Creat
 			return err
 		}
 	} else {
-		err = createImage(rbdVol, volSizeMiB, rbdVol.AdminID, req.GetSecrets())
+		cr, err := util.GetUserCredentials(req.GetSecrets())
```

Comment:
can we reuse the `cr` got in line 114

Reply:
I did not want to pass around `cr` in the `ControllerServer` methods and further the `req` is passed as is to this method, hence left it as is.